myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
Where does Sunday class go it the Visual schedule? If it's on the 7th column of schedule table, it is saying that the weekly schedule ends on Sunday.  Therefore, Monday would be the first day of the week.